api/v2/controllers/channels/channels_controller.py

```diff
@@ -1,10 +1,73 @@
+import os
+import uuid
+
 from flask import make_response, jsonify
-from flask_restful import Resource
+from flask_restful import Resource, request
+from googleapiclient.errors import HttpError
+
+from api.v2.helpers.credentials import check_bouquet_credentials
 from api.v2.helpers.channels.channels_helper import query_all_channels
+from api.v2.models.channels.channels_model import Channels as channels_model
+from api.v2.helpers.bouquets.bouquets_helper import get_bouquet
+from api.v2.utilities.validators import validate_calendar_info
 
 
 class Channels(Resource):
     def get(self):
         channels = query_all_channels()
+        return make_response(jsonify({'channels': channels}), 200)
+
+    def post(self):
+        # Enpoint to add channels to a bouquet
+        try:
+            calendar = request.get_json()
+
+            # Function call to validate channel data
+            is_valid = validate_calendar_info(calendar)
```

Comment:
This variable name `is_valid` denotes a boolean variable yet the return value of `validate_calendar_info` is not boolean. Please correct this.